apps/mobile/src/components/MintLinkButton.tsx

```diff
@@ -113,10 +125,17 @@ export function MintLinkButton({
         icon: <SuperRareIcon width={size === 'sm' ? 16 : 24} height={size === 'sm' ? 16 : 24} />,
       };
     } else if (mintProviderType === 'Highlight') {
-      return {
-        buttonText: 'mint on highlight',
-        icon: <HighlightIcon width={size === 'sm' ? 16 : 24} height={size === 'sm' ? 16 : 24} />,
-      };
+      if (isRadiance && !claimCode) {
+        return {
+          buttonText: 'mint on gallery',
+          icon: <FoundationIcon width={size === 'sm' ? 16 : 24} height={size === 'sm' ? 16 : 24} />,
```

Comment:
lets use the Gallery G icon here

Reply:
my bad realized I didnt like the design in the ticket
we want it to look like:
![CleanShot 2024-04-12 at 11 29 01](https://github.com/gallery-so/gallery/assets/80802871/a1f3d22f-5f57-49d0-94b7-f4b1438a3564)

and inverse on dark mode